pennylane/qnode.py

```diff
@@ -605,8 +605,23 @@ def __call__(self, *args, **kwargs):
             # construct the tape
             self.construct(args, kwargs)
 
-        # execute the tape
-        res = self.qtape.execute(device=self.device)
+        # ************************

```

Comment:
This insertion executes the VQE logic. For now it is in the Qnode, but it will be moved to the device, once the interfaces are moved there too.